### ISSUE TITLE
Is there a way to indicate a skip level for stack in eris ?

### ISSUE THREAD

**Issue by magicalo** [rotisserie/eris#98]
**Is your feature request related to a problem? Please describe.**
I would like to call eris.Wrap inside a utility Logging function but have it attribute the stack to where my function is called vs where .Wrap is called.  (e.g. a typical skip level arg)

**Describe the solution you'd like**
I would prefer to call eris.Wrap inside mypackage.LogError instead of passing it in.  But have eris skip a stack level and report one level above instead of inside my utility function.

So imagine I have mypackage.LogError(eris.Wrap(err, "blah"));
But instead I want to _hide_ the eris.Wrap call inside mypackage.LogError(err, "blah") so that I don't have to litter my code with eris.Wrap everywhere.
I want to tell eris.Wrap to skip the immediate stack frame and record the one above as the frame.

**Describe alternatives you've considered**
Putting    mypackage.LogError(eris.Wrap()) all over my code.

**Additional context**

**Comment by morningvera** [comment]
Thanks for the feature request @magicalo! I can’t think of an automated way of doing this at the moment but I suppose we could add another method for this use case. I need to think about it a bit more but I’ll get back to you soon.

**Comment by magicalo** [comment]
Sounds good.

**Comment by morningvera** [comment]
Yeah, I was thinking of something like that as well. Still trying to think of any alternatives but if I can't think of anything better, I'd be willing to add it.

**Comment by morningvera** [comment]
Hey @magicalo, just letting you know I think I'm going to support this and should have something for you in a few days. Thanks again for the feature request!
